graph-data.rs/src/verify_yaml.rs

```diff
@@ -7,7 +7,11 @@ use std::path::PathBuf;
 use anyhow::Result as Fallible;
 
 pub async fn run() -> Fallible<HashSet<Version>> {
-    let data_dir = PathBuf::from("..");
+    let data_dir = PathBuf::from(".");
+    println!(
+        "Looking for metadata in {:?}",
+        data_dir.canonicalize().unwrap()
```

Comment:
IMO `unwrap` should be a last resort. I think here we can use `?` without an issue.

```suggestion
        data_dir.canonicalize()?
```